test/integration/install_test.go

```diff
@@ -679,6 +693,138 @@ func TestRetrieveUidPostUpgrade(t *testing.T) {
 	}
 }
 
+func TestOverridesSecret(t *testing.T) {
+
+	if TestHelper.GetHelmReleaseName() != "" {
+		t.Skip("Skipping as this is a helm test where linkerd-config-overrides is absent")
+	}
+
+	configOverridesSecret, err := TestHelper.KubernetesHelper.GetSecret(context.Background(), TestHelper.GetLinkerdNamespace(), "linkerd-config-overrides")
+	if err != nil {
+		testutil.AnnotatedFatalf(t, "could not retrieve linkerd-config-overrides",
+			"could not retrieve linkerd-config-overrides\n%s", err.Error())
```

Comment:
`.Error()` not needed
```suggestion
			"could not retrieve linkerd-config-overrides\n%s", err)
```